src/OpenTelemetry.AutoInstrumentation/Configurations/EnvironmentConfigurationTracerHelper.cs

```diff
@@ -241,5 +242,12 @@ public static TracerProviderBuilder AddOtlpExporter(TracerProviderBuilder builde
                 pluginManager.ConfigureTracesOptions(options);
             });
         }
+
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public static TracerProviderBuilder AddAzureInstrumentation(TracerProviderBuilder builder)
+        {
+            AppContext.SetSwitch("Azure.Experimental.EnableActivitySource", true);
+            return builder.AddSource("Azure.*");
```

Comment:
Any chance to be more specific here and provide list of expected DiagnosticSources?
Should we expect lack of any other libraries producing this kind of messages.

Reply:
I don't have an answer regarding whether there will be any exceptions that don't produce the message.

But it looks like it's pretty basic/universal for all Azure SDKs to me because the devblog article in PR description gave a link that the change is made in Azure.Core: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/CHANGELOG.md#1290-2023-03-02